### PR TITLE
Styling articles component with card for each article

### PR DESCRIPTION
**Pull request by adampaulsackfield**

### CODE REVIEW

File: src/components/Articles.jsx [hunk 1]
```diff
@@ -19,10 +24,56 @@ const Articles = () => {
...
 
-			{articles &&
-				articles.map((article) => {
-					return <p>Article: {article.article_id}</p>;
-				})}
+			<div className='grid grid-cols-1 gap-4'>
+				{articles &&
+					articles.map((article) => {
+						const {
+							article_id,
```
Comment (corkesw) [inline src/components/Articles.jsx:31]: article_id is not used so let's not grab it here

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -1,4 +1,9 @@
 import React, { useState, useEffect } from 'react';
+import Moment from 'react-moment';
+// import 'moment-timezone';
+
+// TODO - Links for username, title and buttons for removing and viewing
+
```
Comment (corkesw) [inline src/components/Articles.jsx:6]: You have a catch block in your api function so we will catch the error before we get to the component and never catch it in the component.

**Comment by adampaulsackfield** [comment]
I do plan on using that when I implement the ability to delete an article. Although I can remove it and only add it when I come to use it. I really added all the properties to help me remember which properties I had access too as i abhor switching tabs lol

**Comment by adampaulsackfield** [comment]
That makes sense. I have no got to error handling yet but since I have toasting I may as well catch that error and then show the user. I will do this now and send a new PR.

Thanks

**Comment by corkesw** [comment]
I can't see any styling and the nav bar doesn't collapse for me but I think you are using tailwind? If you are happy with how it looks then all is good! Is Home  going to do something functionally different to Articles  ? If not then maybe it should render a list of all articles using that component - all depends on your wireframe :)

**Comment by adampaulsackfield** [comment]
Thanks. Yes, it appears to be an issue with Tailwindcss and the sandbox you guys are using to view the progress. It does have styling and the hamburger menu works, I did send a video earlier to Jade to confim as she was having the same lack of styling issue. Home was going to house the Articles and Users components, I should really just move the rendering of that component over as you say. I will get that done on the next section.

**Pull request merged**